apps/system/js/screen_manager.js

```diff
@@ -206,6 +206,14 @@
 
         case 'sleep':
           this.turnScreenOff(true, 'powerkey');
+          var lock_enabled = window.lockScreen.enabled;
+          var passcode_enabled = window.lockScreen.passCodeEnabled;
```

Comment:
You can rather listen the 'sleep' SCM event in `lockscreen.js` to access these properties and do the changes you need in that file. Both `request` and events are for communications among different components, and to decouple them by reducing the direct method calls.